Excel-custom-functions/AzureFunction/README.md

```diff
@@ -0,0 +1,234 @@
+    $ npm run start
+    
+    # this will start the server. Then you can sideload the add-in to Excel online using the guidance at https://docs.microsoft.com/office/dev/add-ins/testing/sideload-office-add-ins-for-testing
+    $ npm run start:web
+    ```
+3. In Excel with the add-in loaded, try out the custom function by entering `=CONTOSO.ADD(1,2)` into a cell. The cell should call the custom function, which then calls the Azure function, and returns the result of 3.
+    ![Screenshot of entering the value =CONTOSO.ADD(1,2) into a cell in Excel.](images/contoso-add-function.png)
+
+## Deploy to an Azure function app
+
+You can also deploy and run the Azure function from your own Azure account.
+
+1. Start Visual Studio Code and open the **AzureFunction/AzureFunctionProject** folder in the project.
+2. In Visual Studio Code, press F1 to open the command palette. In the command palette, search for and select **Azure Functions: Deploy to function app....**
+    ![Screnshot of selecting Azure Functions: Deploy to function app in the command palette](images/azure-deploy-task.png)
+    >Note: If you are not signed-in, you are prompted to **Sign in to Azure**. You can also **Create a free Azure account**. After successful sign in from the browser, go back to Visual Studio Code.
+4. Choose **+ Create New Function App in Azure**.
+5. Type a globally unique name that identifies your function app and press Enter. Valid characters for a function app name are `a-z`, `0-9`, and `-`.
+    
+    When you press Enter, the following Azure resources are created in your subscription:
+    
+    - Resource group: Contains all of the created Azure resources. The name is based on your function app name.
+    - Storage account: A standard Storage account is created with a unique name that is based on your function app name.
+    - Hosting plan: A consumption plan is created in the West US region to host your serverless function app.
+    - **Function app**: Your project is deployed to and runs in this new function app.
+    
+    A notification is displayed after your function app is created and the deployment package is applied. Select **View Output** in this notification to view the creation and deployment results, including the Azure resources that you created.
+6. Select the Azure icon on the Activity bar. You should see an Azure Functions area in the Side Bar.
+    ![Screenshot of the Azure icon on the Activity bar.](images/azure-icon.png)
+7. In the **Azure: Functions** area, expand the new function app under your subscription. Expand **Functions**, right-click **AddTwo**, and then choose **Copy function URL**.
+8. You can test that the Azure function is working by calling it from a browser. Open a browser window and enter the URL you just copied. If you want to add numbers, append the following query string to the URL `?first=1&second=2`. You should see the result `{ "answer": 3}` returned to the browser.
```

Comment:
For me, there already was a query parameter on the URL, so I had to append `&first=1&second=2`.